client_plugin/drivers/shared/shared_driver.go

```diff
@@ -145,7 +147,8 @@ func NewVolumeDriver(cfg config.Config, mountDir string) *VolumeDriver {
 	d.dockerd = cli
 
 	// initialize built-in etcd cluster
-	err = d.initEtcd()
+	d.etcd.driver = &d
```

Comment:
Why are we doing this? 
Putting etcdInfo obj in VolumeDriver
And also putting VolumeDriver in etcdInfo obj?